build.gradle.kts

```diff
@@ -44,6 +44,11 @@ subprojects {
       freeCompilerArgs = listOf("-Xopt-in=kotlin.RequiresOptIn")
     }
   }
+  // Ensure "org.gradle.jvm.version" is set to "8" in Gradle metadata.
+  tasks.withType<JavaCompile> {
+    sourceCompatibility = JavaVersion.VERSION_1_8.toString()
```

Comment:
I'm not sure you need `toString()`, but it doesn't really matter.

Reply:
With Kotlin DSL you do, sadly. We need setter overloads.